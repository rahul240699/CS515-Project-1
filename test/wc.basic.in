python3 prog/wc.py basic.txt