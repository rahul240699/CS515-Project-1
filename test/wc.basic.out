7 412 2620 test/WC.1.in